src/abstractdataframe/abstractdataframe.jl

```diff
@@ -2239,3 +2239,36 @@ Base.getindex(::AbstractDataFrame, ::Union{Symbol, Integer, AbstractString}) =
 
 Base.setindex!(::AbstractDataFrame, ::Any, ::Union{Symbol, Integer, AbstractString}) =
     throw(ArgumentError("syntax df[column] is not supported use df[!, column] instead"))
+
+"""
+    reverse(df::AbstractDataFrame)
+
+This function reverses the current order of the rows in `df`.
```

Comment:
Better use a similar formulation as `sort`:
```suggestion
Return a data frame containing the rows in `df` in reversed order.
```